packages/web-components/fast-foundation/src/picker/picker-menu-option.ts

```diff
@@ -0,0 +1,71 @@
+import { attr, html, HTMLView, ViewTemplate } from "@microsoft/fast-element";
+import { FoundationElement } from "../foundation-element";
+
+const defaultContentsTemplate: ViewTemplate<PickerMenuOption> = html`
+    <template>
+        ${x => x.value}
+    </template>
+`;
+
+/**
+ * A picker list item Custom HTML Element.
+ *
+ * @alpha
+ */
+export class PickerMenuOption extends FoundationElement {
+    /**
+     * The underlying string value of the item
+     *
+     * @alpha
+     * @remarks
+     * HTML Attribute: value
+     */
+    @attr({ attribute: "value" })
+    public value: string;
+
+    public contentsTemplate: ViewTemplate;
```

Comment:
Should this be observable? Should there be a contentsTemplateChanged method that calls this.updateView()?

Reply:
yup